check_stability.py

```diff
@@ -1,6 +1,7 @@
 from __future__ import print_function
 
 import argparse
+from ConfigParser import SafeConfigParser
```

Comment:
This change set reverts 0509dfe88e8209a0983b08eed655dfe52804fec2, which seems to have included an unrelated change. I'll push up a second commit that is explicitly dedicated to honoring to the convention you've referenced.